actions/java-gradle-test/action.yaml

```diff
@@ -1,22 +1,19 @@
 name: "Test Java artiacts"
-description: "Run Junit test, publish test results and optionally test signing for Sonatype"
+description: "Run Junit test and publish test results"
 
 inputs:
-  sonar-token:
-    description: "Token for Sonarcloud."
-    required: false
-  sonar-organization:
-    description: "Organization for Sonarcloud."
-    required: false
-  signing-secret-key-ring:
-    description: "Key ring (base64 encoded) for signing the Sonatype publication."
+  download-lfs-files:
+    description: "Whether the Git checkout action should resolve LFS files or not. (Default is false)"
     required: false
-  signing-key-id:
-    description: "Key id for signing the Sonatype publication."
+    default: false
+  gradle-cache:
```

Comment:
Why did you sort this differently? Makes the diff much harder to understand

Reply:
this is because we now try to alphabetically sort our vars in all actions